record-and-playback/notes/scripts/notes.nginx

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
This file has to be renamed - we reverted the rename of the nginx file for etherpad due to the issues it caused. Use something like `recording-notes.nginx`.

Reply:
I reverted the revert that unrenamed the nginx file, it's now `notes-playback.nginx` in the final code.